vendor/github.com/containers/image/openshift/openshift.go

```diff
@@ -136,7 +136,7 @@ func (c *openshiftClient) doRequest(ctx context.Context, method, path string, re
 // getImage loads the specified image object.
 func (c *openshiftClient) getImage(ctx context.Context, imageStreamImageName string) (*image, error) {
 	// FIXME: validate components per validation.IsValidPathSegmentName?
-	path := fmt.Sprintf("/oapi/v1/namespaces/%s/imagestreamimages/%s@%s", c.ref.namespace, c.ref.stream, imageStreamImageName)
+	path := fmt.Sprintf("/apis/image.openshift.io/v1/namespaces/%s/imagestreamimages/%s@%s", c.ref.namespace, c.ref.stream, imageStreamImageName)
```

Comment:
this looks wrong

Reply:
I don't know why the bump picked up the files in github.com or in k8s.io unlike the others I did for 3.10, 3.11, etc.

Reply:
@deads2k merged by you into the 3.9 branch https://github.com/openshift/containers-image/commit/e632a5797c121e35233eedc7cae477844e3d760a. The problem is that we use the 3.9 branch even today.